nautobot_firewall_models/navigation.py

```diff
@@ -4,7 +4,7 @@
 
 menu_items = (
     NavMenuTab(
-        name="Firewall",
+        name="Security",
```

Comment:
everything else is fw, why security here?

Reply:
Previous internal discussion on moving top level to security for subsequent security related plugins to use the same nav menu